apps/ui/app/pages/admin.vue

```diff
@@ -4,13 +4,14 @@ import { computed, ref } from "vue";
 import PageHeader from "../../../tailwind-components/app/components/PageHeader.vue";
```

Comment:
File name should be capitalized (my bad I know, but still).

Reply:
no these are pages and should follow the nuxt convention of lower case file name